server/controllers/StoreController.js

```diff
@@ -0,0 +1,76 @@
+const logger = require('../logger');
+const User = require('../models/user');
+const { StoreItem } = require('../models/store');
+
+// GET /store/items
```

Comment:
Looks great!
Updating the API documentation to reflect these changes is a good idea

Reply:
Change to GET /store for simplicity